tests/pass-dep/libc/libc-epoll-blocking.rs

```diff
@@ -0,0 +1,98 @@
+//@only-target-linux
```

Comment:
Maybe make this set the preemption rate to 0, to ensure the thread interleaving is deterministic?

Reply:
Yup, already did, that's what I am thinking too :D